SOOUM/SOOUM/DesignSystem/Components/SOMCard.swift

```diff
@@ -0,0 +1,338 @@
+//
+//  SOMCard.swift
+//  SOOUM
+//
+//  Created by JDeoks on 9/14/24.
+//
+
+import Foundation
+import UIKit
```

Comment:
import 
```suggestion

import UIKit
```
UIKit 안에 Foundation이 포함되어 있어 `import Foundation` 제거 해주세요